test/common.py

```diff
@@ -2215,6 +2213,7 @@ def make_reftest(self, expected, manually_trigger=False):
       function doReftest() {
         if (doReftest.done) return;
         doReftest.done = true;
+        console.log('doReftest');
```

Comment:
temporary debugging?

Reply:
I'm happy to leave this.. seems useful for debugging.

Reply:
Sounds good.